website/pages/docs/reference/source-spec.md

```diff
@@ -1,8 +1,8 @@
 # Source Spec Reference
 
-Following are available options for the top level source plugin `spec` object. 
+Following are available options for the top level source plugin `spec` object.
 
-Note: For configuring individual plugins, please refer to the configuration section from the relevant plugins from [here](https://www.cloudquery.io/docs/plugins/sources). (e.g. [AWS plugin configuration](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/docs/configuration.md)). Also check out our [recipes](https://www.cloudquery.io/docs/recipes/overview) for common configuration examples.
```

Comment:
This links to 
![image](https://user-images.githubusercontent.com/26760571/204326352-4e774420-3a81-41df-b204-a45deddb8205.png)

so I changed it to link directly to the AWS doc on the Website